CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## [0.4.0] - March 30, 2020
+
+* Migrate the plugin to android v2 embedding.
+* Migrate AndroidX for the example app.
```

Comment:
Thanks for doing this but let's keep the scope of the PR small. This is just about migrating to v2. Let's remove all androidx references.

Reply:
@mehmetf Deleted.